src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX_RANDOM_LENGTH = 100;
+
+    public Figure getRandomFigure() {
+        String color = new ColorSupplier().getRandomColor();
+        Figure[] arrayOfFigures = {new Circle(getRandomLength(), color),
+                new Rectangle(getRandomLength(), getRandomLength(), color),
+                new Square(getRandomLength(), color),
+                new IsoscelesTrapezoid(getRandomLength(),
+                        getRandomLength(), getRandomLength(), color),
+                new RightTriangle(getRandomLength(), getRandomLength(), color)};
```

Comment:
here you create 5 figure objects every time you call the method. Let's first create a random number, and we will create one or another object based on the result